netlogo-headless/src/main/mirror/Mirrorables.scala

```diff
@@ -13,34 +13,38 @@ import org.nlogo.core.AgentVariables.implicitTurtleVariables
 object Mirrorables {
 
   case object Patch extends Kind {
-    object Variables extends Enumeration {
+    object PatchVariables extends Enumeration {
```

Comment:
After reverting this change, the following error shows up in tests:

`scala.UninitializedFieldError: Uninitialized field: /Users/rgg284/IdeaProjects/NetLogo/netlogo-headless/src/main/mirror/Mirrorables.scala: 43 (or 37)`

I'm unsure what's causing this, but it's definitely quite subtle and I can only assume has to do with `lazy val`-style initialization. I've spent about 15 minutes trying to find a minimal case and I've been unable to do so thus far. I'm going back to the old fix and we can revisit this at a later time (since this isn't in `api` or `core` we can introduce breaking changes later).